### ISSUE TITLE
Covariance time-averaging in uvpspec is only correct for thermal noise

### ISSUE THREAD

**Issue by anguta** [HERA-Team/hera_pspec#231]
The covariance averaging in uvpspec currently assumes that samples are uncorrelated in time, baseline, frequency, etc... This is true for thermal noise but will be incorrect for data that is cosmic-variance limited (such as inside of the wedge and measurements of the 21cm signal).

**Comment by philbull** [comment]
@JianrongTan @acliu Has this been solved by your recent changes?

**Comment by JianrongTan** [comment]
I think we don't. We haven't modeled the potential covariance matrix between different time instants and baseline-pairs, while we only tentatively compute the covariance matrix between different delay modes per time and baseline-pair.

**Comment by acliu** [comment]
That's right. @adeliegorce will probably be able to work on this a little bit at some point.
